src/pyspextool/plot/plot_spectra.py

```diff
@@ -5,13 +5,13 @@
 from pyspextool.fit.robust_savgol import robust_savgol
 from pyspextool.plot.limits import get_spec_range
 from pyspextool.io.check import check_parameter, check_file
-from read_spectra_fits import read_spectra_fits
+from pyspextool.io.read_spectra_fits import read_spectra_fits
 from pyspextool.plot import limits
 
 def plot_spectra(file, plot_type='continuous', figsize=None, y='flux',
-                 aperture=0, xlabel=None, ylabel=None, title=None,
+                 aperture=0, xlabel='Wavelength', ylabel='Flux Density', title=None,
                  colors='green', yrange_buffer=0.05, order_numbers=True,
-                 qafileinfo=None):
+                 qafileinfo=None,display=True):
```

Comment:
Is this the name of the keyword you want for all plots like this?  I can fix all my old code if so.

Reply:
could switch to qaplot, but this is not a "QA" plot. I just want to allow it not to display the figure since it is now possible to manipulate the plot object (now an output)